dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -243,9 +247,6 @@ public void setImmediateFlush(boolean immediateFlush) {
         this.immediateFlush = immediateFlush;
     }
 
-    /**
-     * @since 2.0
-     */
```

Comment:
I thought it wasn't all that valuable to have on a validator method which people weren't likely to be calling directly or depending on in their software.

Reply:
I think, this doesn't necessarily have to be there. But since it is already, this provides additional information to our users. The method is `public`, so the JavaDoc would display that the method is available since version 2.0.

Reply:
Yes, it would be great if we could restore that Javadoc comment and then IMHO we're good to go. 🐎

Reply:
Restored, along with a summary line to keep errorprone quiet.

Reply:
Thanks :+1: